static/sass/components/_author.scss

```diff
@@ -0,0 +1,108 @@
+// Author Box
+// ==========
+
+[data-module] {
+  + [data-module='rst-author'] {
+    border-top: 0;
+    margin-top: size('author');
+    padding-top: 0;
+  }
+}
+
+.author-summary {
+  max-width: 100%;
+}
+
+// crop triangle out of svg for birds page
+.bird-face {
+  float: left;
+  height: size('author');
+  margin-right: size('half-shim');
+  overflow: hidden;
+}
+
+.h-card {
+  [data-module='rst-author'] & {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: size('full-content');
+    padding: 0;
+
+    @include above('full-content') {
+      padding: 0 size('gutter');
+      transform: translateX(negative('half-author'));
+    }
+  }
+}
+
+.bio {
+  clear: both;
+}
+
+.bio-short {
+  padding-left: size('half-author');
+  padding-top: size('pixel'); // hack to get margin to to work
+}
+
+[data-size='face'] {
+  color: color('text');
+  float: left;
+  height: size('author-h');
+  width: size('author');
+
+  [data-module='rst-author'] & {
+    shape-outside: polygon(0 0, size('author') 0, size('half-author') size('half-author'));
+    transform: translateY(-65%);
+  }
+}
+
+.socials {
+  // override main ul spacing
+  [data-region='main'] & {
+    margin: 0 0 0 size('half-author');
+    padding-bottom: size('shim');
+    padding-left: 0;
+  }
+
+  li {
+    display: inline-block;
+    list-style: none;
+  }
+
+  a {
+    background-image: none;
+    margin-right: size('shim');
+
+    [data-icon] {
+      padding: size('pixel'); // avoid social icons getting cropped
```

Comment:
if the artboard is 1px of whitespace around the icon we will be ok, but ours typically crop at the artwork. I am not sure of any other way we can avoid it (unless we use px or 100% instead of rem perhaps?)